README.md

```diff
@@ -38,15 +61,15 @@ await decode(image, { fname: 'example', ext: 'jpg' });
 // after creating a directory called 'photos'
 const image = await encode(url, options);
 await decode(image, { fname: './photos/example', ext: 'jpg' });
-```  
+```
 
 ### Contributing
 Read the [CONTRIBUTING](CONTRIBUTING.md) guide for information.  
 
-### License  
+### License
 Licensed under MIT. See [LICENSE](LICENSE) for more information.  
 
-### Issues  
-Report a bug in issues.   
+### Issues
+Report a bug in [issues](https://github.com/riyadhalnur/node-base64-image/issues)
 
-Made with love in Dhaka, Bangladesh by [Riyadh Al Nur](https://verticalaxisbd.com)
+Made with love in Dhaka, Bangladesh by Riyadh Al Nur (https://verticalaxisbd.com)
```

Comment:
Please don't remove links

Reply:
I updated the link format to make the URL directly visible. This way, users can clearly see where the link leads before clicking, which can help build trust and improve transparency.

If you feel otherwise, I can revert that line back ;)

Reply:
Done